tests/viet_kwik_e_mart/custom_features.py

```diff
@@ -0,0 +1,40 @@
+from mindmeld.models.helpers import register_query_feature, register_entity_feature
```

Comment:
This file is not necessary for the vietnamese folder.

Reply:
Done!